ci-operator/step-registry/ipi/deprovision/deprovision/ipi-deprovision-deprovision-commands.sh

```diff
@@ -34,6 +34,9 @@ echo ${SHARED_DIR}/metadata.json
 
 if [[ "${CLUSTER_TYPE}" == "azurestack" ]]; then
   export AZURE_AUTH_LOCATION=$SHARED_DIR/osServicePrincipal.json
+  if [[ -f "${CLUSTER_PROFILE_DIR}/ash-wwt-ca.pem" ]]; then
+    export SSL_CERT_FILE="${CLUSTER_PROFILE_DIR}/ash-wwt-ca.pem"
```

Comment:
Is this because we have no way to directly extract wwt ca file and install it on the local executor system, we have to export the ENV to a custom path?

Reply:
yes, we have no root access in test container to update ca trust, so exported this environment here, else installer will get x509 error as below:
```
level=error msg=failed to fetch Master Machines: failed to load asset "Install Config": failed to create install config: creating Azure session: failed to get Azure environment for the "AzureStackCloud" cloud: Get "https://management.mtcazs.wwtatc.com/metadata/endpoints?api-version=1.0": x509: certificate signed by unknown authority
```

Reply:
Got it.